SAPHana/ra/SAPHanaController

```diff
@@ -166,8 +166,8 @@ function saphana_meta_data() {
     cat <<END
 <?xml version="1.0"?>
 <!DOCTYPE resource-agent SYSTEM "ra-api-1.dtd">
-<resource-agent name="SAPHanaController">
-<version>$SAPHanaControllerVersion</version>
+<resource-agent name="SAPHanaController" version="$SAPHanaControllerVersion">
+<version>1.0</version>
```

Comment:
Just as a question - is this a consequence of clusters also supporting OCF 1.1 standard (in parallel)?
This would also mean a change of other resource agents to. So now <version> is interpreted as OCF version and
we interpreted it as RA version.

Reply:
Correct.

Here's the update to the dev-guide:
https://github.com/ClusterLabs/resource-agents/commit/c5eb4ccf6cee64eeff44513ddc37fbbb0118669b#diff-982679b0ea61ff680d04806236dbb8be613a458f9906b34510bb7db3cc0ac7fd

I've also made a PR for the SAPHanaSR repo.

Reply:
Thanks for the update and info.